pkg/services/rpcsrv/client_test.go

```diff
@@ -1425,7 +1425,7 @@ func TestClient_NNS(t *testing.T) {
 func TestClient_IteratorSessions(t *testing.T) {
 	_, rpcSrv, httpSrv := initServerWithInMemoryChain(t)
 
-	c, err := rpcclient.New(context.Background(), httpSrv.URL, rpcclient.Options{})
+	c, err := rpcclient.New(context.Background(), httpSrv.URL, rpcclient.Options{MaxConnsPerHost: 50})
```

Comment:
Commit description tells about idle connections, and the fix itself restricts `MaxConnsPerHost`. Fix the commit message, add a link to https://github.com/golang/go/issues/20960#issuecomment-447918270.